objects_tdd/spec/lib/cash_register_spec.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/cash_register'
+
+describe CashRegister do
+  describe '#initialize' do
+    it 'should total' do
+      expect(CashRegister.new(0.0)).to be_truthy
+    end
+  end
+
+  describe '#purchase' do
+    it 'should display cost of item' do
+      expect(subject.purchase(1.23)).to eql(1.23)
+    end
+  end
+
+  describe '#total' do
+    it 'should display the current total' do
+      expect(subject.total).to include("Your total is $")
+    end
+  end
+
```

Comment:
how about testing if the total is what you expect after a certain amount has been paid, also maybe a test if the amount of the total is what you expect after purchase is called